content/what-to-avoid.md

```diff
@@ -0,0 +1,32 @@
+# What to avoid
+
+The idea for this page came up during a discussion: quick reference page of
+things that you should not do with Git. Basically, a list of some common Git
+gotchas.
+
+**Postponing commits because the changes are "unfinished"/"ugly"**: It is
+better to have many OK-ish commits than too few perfect commits.  Too few
+perfect commits are probably too large and make undoing more difficult.  It is
+also easier to combine too small commits than it is to split too large commits.
+
```

Comment:
```suggestion

**Not pulling/rebasing before doing new work**: Few things are worse than doing work and then noticing someone else has done it.  Or doing work and noticing a lot of conflicts because un-related work was done in the meantime.
```

Reply:
But this belongs rather in collaborative git? Or should I add it here? (as separate PR? already merged)